ios/TestApp/TestApp/Base.lproj/Main.storyboard

```diff
@@ -0,0 +1,24 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
nit: ideally we want to build UI in code to be able to review it, so any storyboards other than LaunchScreen should be deprecated